app/src/test/java/com/x8bit/bitwarden/ui/vault/feature/vault/util/AccountSummaryExtensionsTest.kt

```diff
@@ -10,6 +10,18 @@ import org.junit.jupiter.api.Test
 
 class AccountSummaryExtensionsTest {
 
+    @Suppress("MaxLineLength")
+    @Test
+    fun `initials should exclude extra spaces and return the starting letters of the first two words for a multi-word name`() {
```

Comment:
Can we put this after the other tests? Or can we combine this with the second test and just add a few spaces after `Second` maybe?

Reply:
I added some spaces before `Second`, after `Second` would always work fine.